README.md

```diff
@@ -3,3 +3,21 @@
 ## https://przemekbartecki.github.io/to-do/
 
 # Korekta projektu graficznego Dominika Gondek
+
+#edit - json server
+
+## Getting started
+
+Install JSON Server 
+
+```
+npm install -g json-server
+```
+
+
+
+Start JSON Server
+
+```bash
+json-server --tasksdb.json
```

Comment:
@PrzemekBartecki komenda to `json-sever tasksdb.json`

Reply:
tak, wkradł się błąd